package.json

```
@@ -105,7 +105,7 @@
     "@shapeshiftoss/hdwallet-native": "1.59.7",
     "@shapeshiftoss/hdwallet-native-vault": "1.59.7",
     "@shapeshiftoss/hdwallet-phantom": "1.59.7",
-    "@shapeshiftoss/hdwallet-walletconnectv2": "1.59.7",
+    "@shapeshiftoss/hdwallet-walletconnectv2": "1.59.8-alpha.1",
```

Update me once https://github.com/shapeshift/hdwallet/pull/719 is published.

```suggestion
    "@shapeshiftoss/hdwallet-walletconnectv2": "1.59.8",
```